### DISCUSSION TITLE
How would I right_format at different levels than cursor?

### DISCUSSION THREAD

**Discussion by JxJxxJxJ** [starship/starship#4795] [Q&A] [answered]
For example, I have the following 
![image](https://user-images.githubusercontent.com/103049321/211121849-678aa494-42e8-41e3-bbb7-0d84127bc64c.png)
As you can see, it would make sense I could make the `cmd_duration` one line higher, but I'm struggling finding how to do that, any suggestions?

**Comment by matchai** [comment, marked as answer]
I'd recommend taking a look at the [`fill` module](https://starship.rs/config/#fill), which would be able to be used as a spacer for your first prompt line.

**Reply by JxJxxJxJ** [reply]
What if I'm using [username] [directory] as different modules? In `~/.config/starship.toml`

> format = 'AA $fill BB $fill CC'
> 
> [fill]
> symbol = '-'
> style = 'bold green'

`AA`, `BB`, `CC` are not modules, are they?

**Reply by matchai** [reply]
In the provided example, `AA`, `BB`, and `CC` are static strings, but they could be replaced with modules, like so:
```toml
format = '$directory$fill$username'
```

You can read further about the root-level `format` option, here: https://starship.rs/config/#prompt

**Reply by JxJxxJxJ** [reply]
Thank you so much! That helped!
![image](https://user-images.githubusercontent.com/103049321/211213788-d1f6d18f-dba5-4670-8060-c483daf61d58.png)
